polylogue/cli/click_app.py

```
@@ -129,8 +171,20 @@ def _run_click_help(env: CommandEnv, ctx: click.Context, topic: Optional[str], e
-    use_plain = _should_use_plain(plain=plain, interactive=interactive)
-    ctx.obj = _build_env(use_plain)
+    _ensure_required_tools()
+    _ensure_required_modules()
+    use_plain, plain_reason = _should_use_plain(plain=plain, interactive=interactive)
+    env = _build_env(use_plain)
+    ctx.obj = env
+    if use_plain and plain_reason and plain_reason != _PLAIN_REASON_FLAG:
+        _announce_plain_mode(env, plain_reason)
+ 
+ 
+def _announce_plain_mode(env: CommandEnv, reason: str) -> None:
```

Ruff (0.15.9)

[warning] Unused function argument: `env`

(ARG001)